runnc-cli/Makefile

```diff
@@ -6,6 +6,7 @@ build: runnc
 runnc:
 	GOOS=linux GOARCH=amd64 go build -o $@ .
 
+# TODO: Currently install involves copying runnc-cont and nabla-run into /opt/runnc/bin/
```

Comment:
didn't get this. Is the intent to have both the old runnc and the new runnc2?

Reply:
The TODO is there since it was not pretty to copy `runnc-cont` and `nabla-run` from a higher level directory. Mainly there as a reminder to include these when we replace the original `runnc`.